frontend/src/Context/SessionProvider/session.tsx

```diff
@@ -146,7 +118,8 @@ const MAX_PIN_COUNT = isMobile() ? MAX_PIN_COUNT_MOBILE : MAX_PIN_COUNT_DESKTOP;
  * @returns {SessionContextType} a context provider for a publisher preview
  */
 const SessionProvider = ({ children }: SessionProviderProps): ReactElement => {
-  const session = useRef<Session | null>(null);
+  const [, forceUpdate] = useState<boolean>(false); // NOSONAR
```

Comment:
Takes the place of the `setChangedStream` state function. Without a re-render during a stream change, we don't get visual indicators for a subscriber muting themselves or the initials being displayed.

Reply:
Would be good to add this context somewhere 🙏